gm4_animi_shamir/beet.yaml

```diff
@@ -2,19 +2,22 @@ extend: '../module.yaml'
 
 id: gm4_animi_shamir
 name: Animi Shamir
+version: 1.0.X
 
 pipeline:
   - data_pack:
       load: .
 
 meta:
   gm4:
-    description: Adds the Animi Shamir to Metallurgy. Items with Animi will respawn with you when you die!
     required:
-      - gm4_metallurgy
-    recommended: []
-    notes:
-      - Modifies items right after a player died, and might fight over those items with other Datapacks that do the same.
+      - 'gm4_metallurgy: 1.2.0'
```

Comment:
Is it considered, proper? to have the yaml keys be predetermined? Like the module here listed becomes the config option rather than the value. 

It's possible I just don't know the conventions.